Examples/KitbotDemoBasic/src/main/java/frc/robot/Subsystems/DrivetrainSubsystem.java

```diff
@@ -32,14 +32,14 @@ private void setVoltages(double left, double right) {
   }
 
   public CommandBase setVoltagesCommand(DoubleSupplier left, DoubleSupplier right) {
-    return new RunCommand(() -> this.setVoltages(left.getAsDouble(), right.getAsDouble()), this);
+    return this.run(this.setVoltages(left.getAsDouble(), right.getAsDouble()));
   }
 
   public CommandBase setVoltagesArcadeCommand(DoubleSupplier drive, DoubleSupplier steer) {
-    return new RunCommand(() -> {
+    return this.run(() -> {
       var speeds = DifferentialDrive.arcadeDriveIK(drive.getAsDouble(), steer.getAsDouble(), false);
       this.setVoltages(speeds.left * 12, speeds.right * 12);
-    }, this);
+    });
```

Comment:
These need to be reverted, yeah? `setVoltages` is void.

Reply:
`this.run` replaces the `this` at the end of the `RunCommand` constructor